src/main/java/lingogo/commons/util/FileUtil.java

```diff
@@ -96,4 +113,78 @@ public static boolean isValidCsvFileName(String fileName) {
                 && fileName.length() < 32;
     }
 
+    /**
+     * Returns a List of String[] representing the CSV file content at the specified
+     * filepath. Each element of the List contains a row of data in the CSV file.
+     *
+     * @param filepath A string representing the file path. Cannot be null.
+     * @param expectedHeaders The expected headers in the CSV file.
+     * @throws IOException If an error occurs during the read.
+     * @throws CsvValidationException When the CSV file is not valid.
+     * @throws CsvColumnHeaderException When the headers of the CSV file do not match expectedHeaders.
+     * @throws CsvNumColumnsException When the number of columns in the CSV file does not match the
+     * length of expectedHeaders.
+     */
+    public static List<String[]> importCsvFileContent(String filepath, String[] expectedHeaders)
+            throws IOException, CsvValidationException, CsvColumnHeaderException, CsvNumColumnsException {
+        CSVReader reader = new CSVReaderBuilder(
```

Comment:
`reader` needs to be closed regardless of whether an exception is thrown or not, else file name cannot be modified or the data directory cannot have its name changed/deleted. Can consider using try-with-resources to declare the `CSVReader` so that it will automatically close regardless of an exception being thrown (see Jae's previous code).